src/Dapr.Actors/Runtime/Actor.cs

```diff
@@ -298,7 +298,7 @@ protected Task UnregisterReminderAsync(string reminderName)
                 timerName = $"{this.Id}_Timer_{this.timers.Count + 1}";
             }
 
-            var actorTimer = new ActorTimer(this, timerName, asyncCallback, state, dueTime, period);
```

Comment:
There's no need for the timer to have a backreference to the actor - it was unused.